.github/workflows/cont_integration.yml

```diff
@@ -75,7 +77,7 @@ jobs:
             ~/.cargo/registry
             ~/.cargo/git
             target
-          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.toml','**/Cargo.lock') }}
+          key: ${{ runner.os }}-cargo-${{ github.job }}-${{ hashFiles('**/Cargo.toml','**/Cargo.lock') }}
```

Comment:
from the jobs output it looks `github.job` resolves just to `CI` is it useful

Reply:
I think that was with `github.workflow`, I was using it previously. Then i switched to `github.job` and force-pushed.. maybe you were looking at the log from the previous run

Reply:
oh, yes